### DISCUSSION TITLE
Nuxt 3 Nested route scroll behaviour seems weird

### DISCUSSION THREAD

**Discussion by chunglam2525** [nuxt/nuxt#22191] [General]
Demo: https://stackblitz.com/edit/github-6hjnum-7eeqyv?file=app.vue
nuxt: 3.6.3

I have a demo here using app/router.option.ts to create my desired scrolling behaviour, where the scroll position will stay still when navigating between nested route.
But shouldn't this be the default behaviour or a toggle? Without me having to create such a complex checking.

**Comment by ojvribeiro** [comment]
I think there should be at least a option to toggle whether it should scroll to top or not in `<NuxtPage>` and/or `navigateTo` function.

**Comment by awacode21** [comment]
any new insights?
